src/components/wishlists/WishlistCard.jsx

```diff
@@ -0,0 +1,46 @@
+import { useCart } from "../../context/cart-context";
+import { getDiscountOfProduct } from "../utilities/getDiscountOfProduct";
+
```

Comment:
have a consistency about component export keep it named I would day  
Also either use ES6 or not don't mix them if arrow they maintain that don't have function there